9.1/paper.tex

```diff
@@ -478,10 +484,34 @@ \subsection{Full support for $hp$~adaptivity in parallel computations}
 
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\subsection{Interfaces to the HDF5 file format and libraries}
+\subsection{Interface to the HDF5 file format and libraries}
```

Comment:
Can you also add an introductory sentence that explains what HDF5 is?

Reply:
I added an introductory paragraph of HDF5, is that ok?